webpack/assets/javascripts/react_app/components/SettingUpdateModal/components/SettingForm/SettingForm.js

```diff
@@ -18,7 +18,7 @@ const SettingForm = ({
     let submitValues = { setting: values };
 
     if (setting && setting.settingsType === 'array') {
-      const splitValue = { value: values.value.split(',') };
+      const splitValue = { value: values.value === '' ? [] : values.value.split(',') };
```

Comment:
In most languages `''.split(',')` returns `[]` but looks like Javascript is weird.